offix/src/main/java/org/aerogear/offix/Offix.kt

```diff
@@ -182,6 +237,37 @@ fun getDao() = Offline.getDb()?.mutationDao()
  */
 fun inputTypeChecker(string: String) = string.equals("com.apollographql.apollo.api.Input")
 
+/*
+ This interceptor should be used by the user while making an instance of apollo client
+ which will help in detecting conflicts.
+ */
+fun getResponseInterceptor(): Interceptor? {
+
+    val conflictInterceptor = Interceptor {
+        val request = it.request()
+
+        val response = it.proceed(request)
+
+        //https://stackoverflow.com/a/33862068/10189663
+        val responseBody = response.body()
+        val bufferedSource = responseBody?.source()
+        bufferedSource?.request(Long.MAX_VALUE)
+        val buffer = bufferedSource?.buffer()
+        val responseBodyString = buffer?.clone()?.readString(Charset.forName("UTF-8")) ?: ""
+
+        Log.d("OffixClass", " Interceptor ** : $responseBodyString")
+
+        //To see for conflict, "VoyagerConflict" which comes in the message is searched for.
+        if (responseBodyString.contains("VoyagerConflict")) {
```

Comment:
Not going to work. Please check for specific payload this name is not part of the specification!

Reply:
Okay, I would be detecting then on the basis of null data returned by the server in onResponse() method.